webots_ros2_universal_robot/launch/multirobot_launch/world_multirobot_launch.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+# Copyright 1996-2021 Cyberbotics Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Launch Webots Universal Robot and ABB Robot simulation world."""
+
+import launch
+from launch import LaunchDescription
+from launch.substitutions import LaunchConfiguration
+from launch.actions import DeclareLaunchArgument
+from launch.substitutions.path_join_substitution import PathJoinSubstitution
+from ament_index_python.packages import get_package_share_directory
+from webots_ros2_driver.webots_launcher import WebotsLauncher, Ros2SupervisorLauncher
+
+
+PACKAGE_NAME = 'webots_ros2_universal_robot'
+
+
+def generate_launch_description():
+    package_dir = get_package_share_directory(PACKAGE_NAME)
+    world = LaunchConfiguration('world')
+
+    webots = WebotsLauncher(
+        world=PathJoinSubstitution([package_dir, 'worlds', world])
+    )
+
+    ros2_supervisor = Ros2SupervisorLauncher()
+
+    return LaunchDescription([
+        DeclareLaunchArgument(
+            'world',
+            default_value='armed_robots.wbt',
```

Comment:
What about this name?
```suggestion
            default_value='robotic_arms.wbt',
```